src/main/java/org/gridsuite/modification/server/repositories/LineCatalogRepository.java

```diff
@@ -0,0 +1,24 @@
+/*
+  Copyright (c) 2023, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.repositories;
+
+import org.gridsuite.modification.server.LineKind;
+import org.gridsuite.modification.server.entities.LineTypeEntity;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * @author Sylvain Bouzols <sylvain.bouzols at rte-france.com>
+ */
+@Repository
+public interface LineCatalogRepository extends JpaRepository<LineTypeEntity, UUID> {
+
+    List<LineTypeEntity> findAllByKind(LineKind kind);
```

Comment:
findAllByLineKind
By convention for query method, after `By` is the property name

Reply:
Yep, "kind" is the property name, "LineKind" is the property type, I think it's ok.
It's not even a naming convention, it can't work at all if it's not a property name after "By".